modules/gdscript/tests/scripts/parser/warnings/static_called_on_instance.gd

```diff
@@ -0,0 +1,11 @@
+class Player:
+	var x = 3
+
+func test():
+	# These should not emit a warning.
+	var _player = Player.new()
+	print(String.num_uint64(8589934592)) # 2 ^ 33
+
+	# This should emit a warning.
```

Comment:
Didn't you mean to also test standalone `Player.new()` to match the "no warning" case?

Reply:
``_player.new()`` emits a totally different kind of error so I figured it was better left out